geary-commons-papermc/src/main/kotlin/com/mineinabyss/geary/minecraft/actions/IgniteBlocksInRadiusAction.kt

```diff
@@ -0,0 +1,41 @@
+package com.mineinabyss.geary.minecraft.actions
+
+import com.mineinabyss.geary.ecs.api.actions.GearyAction
+import com.mineinabyss.geary.ecs.api.entities.GearyEntity
+import org.bukkit.Location
+import org.bukkit.Material
+import org.bukkit.entity.Entity
+
+class IgniteBlocksInRadiusAction(
+    private val radius: Double  // Again not really a radius but close enough
+) : GearyAction() {
+    val GearyEntity.entity by get<Entity>()
+
+    override fun GearyEntity.run(): Boolean {
+        val intRadius = radius.toInt()
+        for (x in -intRadius..intRadius) {
+            for (y in -intRadius..intRadius) {
+                for (z in -intRadius..intRadius) {
+                    val location = Location(
+                        entity.world,
+                        entity.location.x + x,
+                        entity.location.y + y,
+                        entity.location.z + z
+                    )
+                    if (location.block.isEmpty) {
+                        val belowLocation = Location(
```

Comment:
use `location.add(0.0, 1.0, 0.0)`

Also why's it called belowLocation if we are adding one? would it not be above?

Reply:
Yeah mistype